feature-libs/product-configurator/rulebased/components/group-menu/configurator-group-menu.component.ts

```diff
@@ -290,4 +285,56 @@ export class ConfiguratorGroupMenuComponent {
       })
     );
   }
+
+  /**
+   * Prevents page down behaviour when users press 'ArrowUp' or 'ArrowDown' keys
+   * to change the focus to the corresponding group
+   *
+   * @param {KeyboardEvent} event - Keyboard event
+   */
+  switchGroupOnArrowPress(
+    event: KeyboardEvent,
+    groupIndex: number,
+    group?: Configurator.Group
+  ): void {
+    if (event.code === 'ArrowUp' || event.code === 'ArrowDown') {
+      this.configGroupMenuService.switchGroupOnArrowPress(event, groupIndex);
+    } else if (event.code === 'ArrowLeft' || event.code === 'ArrowRight') {
```

Comment:
Why both on arrow left an right we perform going deeper in the tree or going up the tree? 
[opinion] I'd expect arrow left to always let me navigate out (up) in the tree. But arrow right to navigate deeper (if possible). But it's just a personal opinion. Please verify standards with @developpeurweb. 

If I'm right above, then I perhaps we should swap the meaning of arrow left and right for RTL storefronts. Please verify with @developpeurweb too.

Reply:
Done! I took your feedback into account regarding LTR / RTL direction